### ISSUE TITLE
Some sites dosn't like autofill

### ISSUE THREAD

**Issue by zynexiz** [marius-wieschollek/passwords-webextension#74]
Om not sure what causes this, if there's another way some sites handle the inputs. When trying to click the password for some sites, it doesn't recognize the input the extension sends to the input field (NextCloud forum is one of them). It just shows an error, or act if the input was blank. I seen this problem om some sites.

The solution for now is to manually copy the username and the password and paste it to the input field. May there is some odd java code involved in this, haven't debugged the error itself.

It might be related to https://github.com/marius-wieschollek/passwords-webextension/issues/32

**Comment by marius-wieschollek** [comment]
If you want to, you can try if this still happens with the experimental version 2.0. 

You can download it for [Firefox here](https://git.mdns.eu/nextcloud/passwords-webextension/-/jobs/artifacts/testing/browse/dist/firefox-extension?job=Publish+Nightly+Firefox) or [Chrome here](https://chrome.google.com/webstore/detail/aipncmjoigmhooiiclcailmhiopachih). I would recommend Firefox, because the version in the Chrome Web Store always lags behind by some days.

**Comment by zynexiz** [comment]
I just tried it on Firefox for Android, and nothing showed up when opening the addon, had to reinstall it. I will try it on a normal desktop later.

**Comment by marius-wieschollek** [comment]
I think i had the same issue today as well. In that case it should be fixed in 2.0.0.3435

**Comment by zynexiz** [comment]
Just tested 2.0.0.3435, and did not get it to work. I got the UI at first (very nice looking btw!), but I couldn't use the account I had added before. If I tried to delete it it said 'Function not implemented yet', same if I tried to save. The token buttons didn't work either. So I couldn't test it. After I removed the addon completely, I didn't see the UI anymore.

This was on Firefox 73.0, Arch Linux.

**Comment by marius-wieschollek** [comment]
These issues should now be fixed. Accounts can be edited in the settings, including the token.

Upgrading vrom v1.x should no longer fail, even if it was never set up completely.

**Comment by lightlike** [comment]
Issue is still present.
I am using the developer version of passwords. (v2.0.0.3501, Chrome, Win10)
Does not work with pages like:

- OpenMediaVault 5 web interface
- fritz.box web interface
- OpenVPN web interface
- plex.tv
- plex WebTools plugin
- Syncthing Web (but this is a special case as it shows a browser dialog)
- and some more i do not remember

Most of the time this seems to be an input-field with *type*, *name* or *id* set to "password" or something containing "password". (I can past some code for the sides you might not have. If needed)
I do not know if any of those worked before as I am pretty new to this.

**Comment by majkinetor** [comment]
I had the same problem, fix was to add space in password field and delete it :)
However v2 solve it for my cases.

**Comment by marius-wieschollek** [comment]
Does this issue still exist with v2 of the extension?

**Comment by lightlike** [comment]
I updated 1 or 2 weeks ago and this is a lot better. Works on almost all pages I use it on (even the wierd sharepoint-"thing" from university).

Up until now I only found 2 instances where this does not work.
1. openmediavault (that one seems to behave a bit wierd)
2. authentication using a popup (see syncthing below as an example)

But regarding everything else: very nice work.


![temp](https://user-images.githubusercontent.com/12970722/103588704-182f5100-4eea-11eb-831b-19c15ca48435.PNG)
